fuel/config_parser.py

```diff
@@ -51,6 +51,11 @@
    The default :class:`~numpy.dtype` to use for floating point numbers. The
    default value is ``float64``. A lower value can save memory.
 
+.. option:: local_data_path
+
+   The local path where the dataset is going to be copied. This is a useful
```

Comment:
Could you please elaborate that the copying will only take place at the first time, and after that the cached copy will be used?